components/script/dom/htmltableelement.rs

```diff
@@ -398,16 +398,20 @@ impl HTMLTableElementMethods for HTMLTableElement {
     // https://html.spec.whatwg.org/multipage/#dom-table-deleterow
     fn DeleteRow(&self, mut index: i32) -> Fallible<()> {
         let rows = self.Rows();
+
         // Step 1.
```

Comment:
Traditionally just the step numbers were included in comments, but it turns out that the number changes a lot so a lot of these comments have become misleading. I think it's a good practice now to include the actual step text as it makes the code more future proof.

```suggestion
        // Step 1: If index is less than −1 or greater than or equal to the number of elements
        // in the rows collection, then throw an "IndexSizeError".
```